docs/source/operations/projections/patterson.rst

```diff
@@ -0,0 +1,46 @@
+.. _patterson:
+
+********************************************************************************
+Patterson
+********************************************************************************
+
+The Patterson projection is a new cylindrical map projection designed for
```

Comment:
Drop the "new" - According to the reference it is at this time already 4 years old. I don't know when the shift from new to old/established happens but I am certain that we won't remember to adjust the documentation when it happens :-) Let's just stick to the facts here.

Reply:
Done